Rakefile

```diff
@@ -47,7 +49,7 @@ namespace :import do
 
     datasets.each do |country, year|
       dest = Pathname.new("datasets/#{ country }")
-      csvs = Pathname.glob("../etdataset/data/#{ country }/#{ year }/*/output/*.csv")
+      csvs = Pathname.glob("#{ ENV['ETDATASET_PATH'] }/data/#{ country }/#{ year }/*/output/*.csv")
```

Comment:
Pass the dataset path to `encrypt_balance` as an argument rather than accessing the global variable directly. This means that we can also check in the Rake task that a value has been set.